modules/nodes-nodes-garbage-collection-configuring.adoc

```diff
@@ -147,6 +136,8 @@ $ oc create -f gc-container.yaml
 kubeletconfig.machineconfiguration.openshift.io/gc-container created
 ----
 
+.Verification
+
 . Verify that garbage collection is active. The Machine Config Pool you specified in the custom resource appears with `UPDATING` as 'true` until the change is fully implemented:
```

Comment:
Beyond the scope of this PR, but...

Verify that garbage collection is active by entering the following command.